### PR TITLE
Print bidings for CPNs

### PR DESCRIPTION
**Pull request by srba**
Added the posibility to print bindings for CPNs whenever the trace option is different from NONE.

### CODE REVIEW

File: include/PetriEngine/Colored/Unfolder.h [hunk 1]
```diff
@@ -61,15 +61,18 @@ namespace PetriEngine {
...
+            
         public:
-            Unfolder(const ColoredPetriNetBuilder& b, const PartitionBuilder& partition, const VariableSymmetry& symmetry, const ForwardFixedPoint& fixed_point)
+            Unfolder(const ColoredPetriNetBuilder& b, const PartitionBuilder& partition, const VariableSymmetry& symmetry, const ForwardFixedPoint& fixed_point, bool print_bindings)
             : _builder(b),
               _stable(b),
               _symmetry(symmetry),
               _partition(partition),
-              _fixed_point(fixed_point) {}
+              _fixed_point(fixed_point) {_print_bindings=print_bindings; }
```
Comment (petergjoel) [inline include/PetriEngine/Colored/Unfolder.h:75]: Change to
`_fixed_point(fixed_point), _print_bindings(print_bindings) {}`

File: src/main.cpp [hunk 2]
```diff
@@ -159,7 +159,7 @@ int main(int argc, const char** argv) {
             options.computePartition, options.symmetricVariables,
             options.computeCFP, out,
             options.partitionTimeout, options.max_intervals, options.max_intervals_reduced,
-            options.intervalTimeout, options.cpnOverApprox);
+            options.intervalTimeout, options.cpnOverApprox, options.trace != TraceLevel::None);
```
Comment (petergjoel) [inline src/main.cpp:162]: I would recommend making a new option.

**Review by petergjoel — APPROVED** [review]
Small changes, should be okay as is.

File: src/main.cpp (hunk 2 above)
Comment (srba) [reply src/main.cpp]: I have introduced a new switch option -b --bindings

**Pull request merged**